widgetssdk/src/testSnapshot/java/com/glia/widgets/snapshotutils/SnapshotPicasso.kt

```diff
@@ -64,7 +91,7 @@ interface SnapshotPicasso: SnapshotTestLifecycle {
     ): Int {
         val imageIndex = index % imageResources.size
         val imageResource = imageResources[imageIndex]
-        imageView.setImageResource(imageResource)
+//        imageView.setImageResource(imageResource)
```

Comment:
I think it might be good to add a comment why we have this line disabled and when we might need to enable it.

Reply:
Or just remove it entirely

Reply:
Thank you for your attention!